{"uid":"7f7a46f21950d9ab","name":"test_full_cluster_restart[SIGTERM-postgres]","fullName":"tests.integration.ha_tests.test_self_healing#test_full_cluster_restart","historyId":"1fae0f26d7819f653f6b98b8e1174bd5","time":{"start":1727747670655,"stop":1727747670655,"duration":0},"description":"This tests checks that a cluster recovers from a full cluster restart.\n\n    The test can be called a full cluster crash when the signal sent to the OS process\n    is SIGKILL.\n    ","descriptionHtml":"<p>This tests checks that a cluster recovers from a full cluster restart.</p>\n<pre><code>The test can be called a full cluster crash when the signal sent to the OS process\nis SIGKILL.\n</code></pre>\n","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 840, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 875, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 697, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 532, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 617, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1091, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"tmp_path_factory","time":{"start":1727746245976,"stop":1727746245976,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop","time":{"start":1727746245976,"stop":1727746245976,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1727746245977,"stop":1727746246131,"duration":154},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1727747670655,"stop":1727747670655,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"ops_test::finalizer","time":{"start":1727747670752,"stop":1727747671418,"duration":666},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1727747671418,"stop":1727747671419,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1727747671419,"stop":1727747671419,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1727747671419,"stop":1727747671419,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_self_healing"},{"name":"host","value":"fv-az1148-229"},{"name":"thread","value":"15806-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_self_healing"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"process","value":"'postgres'"},{"name":"signal","value":"'SIGTERM'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"eb1207cb85cb36df","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":17,"broken":2,"skipped":55,"passed":35,"unknown":6,"total":115},"items":[{"uid":"9465a4fc6a48bd9f","reportUrl":"../190//#testresult/9465a4fc6a48bd9f","status":"passed","time":{"start":1727661662303,"stop":1727661733255,"duration":70952}},{"uid":"9070a64a483a0d72","reportUrl":"../189//#testresult/9070a64a483a0d72","status":"passed","time":{"start":1727575435636,"stop":1727575515676,"duration":80040}},{"uid":"a6f2adc61704d8ea","reportUrl":"../188//#testresult/a6f2adc61704d8ea","status":"passed","time":{"start":1727488945709,"stop":1727489015857,"duration":70148}},{"uid":"d2e0f2ba23a256b9","reportUrl":"../187//#testresult/d2e0f2ba23a256b9","status":"passed","time":{"start":1727402440982,"stop":1727402511003,"duration":70021}},{"uid":"1cecd1fd13e23990","reportUrl":"../186//#testresult/1cecd1fd13e23990","status":"passed","time":{"start":1727316137188,"stop":1727316224097,"duration":86909}},{"uid":"8c87b0af8f1ba701","reportUrl":"../184//#testresult/8c87b0af8f1ba701","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1727228827251,"stop":1727228827251,"duration":0}},{"uid":"ab9bd8a84b13d48b","reportUrl":"../183//#testresult/ab9bd8a84b13d48b","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1727142516500,"stop":1727142516500,"duration":0}},{"uid":"31be19778f432f61","reportUrl":"../182//#testresult/31be19778f432f61","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1727056073128,"stop":1727056073128,"duration":0}},{"uid":"90337a5fbbc74728","reportUrl":"../181//#testresult/90337a5fbbc74728","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726969742063,"stop":1726969742063,"duration":0}},{"uid":"e053a512e7c74bac","reportUrl":"../180//#testresult/e053a512e7c74bac","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726883241546,"stop":1726883241546,"duration":0}},{"uid":"49efb5f5b2c5b39a","reportUrl":"../179//#testresult/49efb5f5b2c5b39a","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726796841456,"stop":1726796841456,"duration":0}},{"uid":"4576dbe13108e9fe","reportUrl":"../178//#testresult/4576dbe13108e9fe","status":"unknown","time":{}},{"uid":"be2822a972021e5e","reportUrl":"../177//#testresult/be2822a972021e5e","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726624020839,"stop":1726624020839,"duration":0}},{"uid":"14cc0dbeeac22964","reportUrl":"../176//#testresult/14cc0dbeeac22964","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726537915407,"stop":1726537915407,"duration":0}},{"uid":"86ff3b8eec42a362","reportUrl":"../175//#testresult/86ff3b8eec42a362","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726451580526,"stop":1726451580526,"duration":0}},{"uid":"758f87ddcec186e","reportUrl":"../174//#testresult/758f87ddcec186e","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726365197152,"stop":1726365197152,"duration":0}},{"uid":"b6438a05a4549df4","reportUrl":"../173//#testresult/b6438a05a4549df4","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726278765448,"stop":1726278765448,"duration":0}},{"uid":"4c50414a28837366","reportUrl":"../172//#testresult/4c50414a28837366","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726192392060,"stop":1726192392060,"duration":0}},{"uid":"bb9c4ed3f057a0d","reportUrl":"../171//#testresult/bb9c4ed3f057a0d","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726105962537,"stop":1726105962537,"duration":0}},{"uid":"6c357fbfa720150c","reportUrl":"../170//#testresult/6c357fbfa720150c","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726019567115,"stop":1726019567115,"duration":0}}]},"tags":["asyncio","abort_on_fail","group(1)"]},"source":"7f7a46f21950d9ab.json","parameterValues":["'postgres'","'SIGTERM'"]}